portfolio/src/main/webapp/script.js

```diff
@@ -15,14 +15,4 @@
 /**
  * Adds a random greeting to the page.
  */
-function addRandomGreeting() {
```

Comment:
Let's keep this for now as I think there might be some javascript required next week and we can always repurpose this function to do something else.